org/mozilla/jss/ssl/javax/JSSEngineReferenceImpl.java

```diff
@@ -196,31 +196,31 @@ private void createBufferFD() {
         // Turn on SSL Alert Logging for the ssl_fd object.
         int ret = SSL.EnableAlertLogging(ssl_fd);
         if (ret == SSL.SECFailure) {
-            throw new RuntimeException("JSSEngine.init(): Unable to enable SSL Alert Logging on this SSLFDProxy instance.");
+            throw new SSLException("Unable to enable SSL Alert Logging on this SSLFDProxy instance.");
         }
 
         ret = SSL.EnableHandshakeCallback(ssl_fd);
         if (ret == SSL.SECFailure) {
-            throw new RuntimeException("JSSEngine.init(): Unable to enable SSL Handshake Callback on this SSLFDProxy instance.");
+            throw new SSLException("Unable to enable SSL Handshake Callback on this SSLFDProxy instance.");
         }
 
         // Pass this ssl_fd to the session object so that we can use
         // SSL methods to invalidate the session.
     }
 
-    private void initClient() {
+    private void initClient() throws SSLException {
         debug("JSSEngine: initClient()");
 
         if (cert != null && key != null) {
             debug("JSSEngine.initClient(): Enabling client auth: " + cert);
             ssl_fd.SetClientCert(cert);
             if (SSL.AttachClientCertCallback(ssl_fd) != SSL.SECSuccess) {
-                throw new RuntimeException("JSSEngine.init(): Unable to attach client certificate auth callback.");
+                throw new SSLException("Unable to attach client certificate auth callback.");
             }
         }
     }
 
-    private void initServer() {
+    private void initServer() throws SSLException, IllegalArgumentException {
```

Comment:
`IllegalArgumentException` is a `RuntimeException` so it doesn't have to be declared.